### DISCUSSION TITLE
Issue while signing for github student develoer pack

### DISCUSSION THREAD

**Discussion by AdharshJolly** [community/community#42687] [GitHub Education]
I tried to sign up for the developer pack and when I upload the id proof required and click on the proceed button I am asked to complete my profile but my profile is already complete yet this issue persists

**Comment by pierizvi** [comment]
Use a **different browser** , if you are using  **Brave** close the adblocker or see this reference  #41026  it might help .

**Comment by NarpatAanjana** [comment]
Hello, if you try "https://education.github.com/pack"   by this site then wait for a week you get mail
otherwise, fill this website
